youtube_dl/extractor/streamcz.py

```diff
@@ -93,4 +93,15 @@ def _real_extract(self, url):
             'description': data.get('web_site_text'),
             'duration': int_or_none(data.get('duration')),
             'view_count': int_or_none(data.get('views')),
+            'subtitles': self.extract_subtitles(data.get('subtitles_srt')),
         }
+
+    @staticmethod
+    def _get_subtitles(subs):
```

Comment:
It's the method called by `self.extract_subtitles()` common method when subtitles are actually requested by user.

In this case, it's probably unnecessary.